pkg/sabakan/main.go

```diff
@@ -178,6 +192,25 @@ func subMain(ctx context.Context) error {
 	}
 	s.ListenAndServe()
 
+	//sabakan TLS
+	counter = metrics.NewCounter()
+	webServerHTTPS := web.NewServer(model, cfg.IPXEPath, cryptsetupPath, advertiseURL, advertiseURLHTTPS, allowedIPs, cfg.Playground, counter, true)
+	ss := &well.HTTPServer{
+		Server: &http.Server{
+			Addr:    cfg.ListenHTTPS,
+			Handler: webServerHTTPS,
+		},
+		ShutdownTimeout: 3 * time.Minute,
+		Env:             env,
+	}
+	err = ss.ListenAndServeTLS(cfg.SabakanTLSCertFile, cfg.SabakanTLSKeyFile)
```

Comment:
If we update sabakan before creating cert files, all of the sabakan API fails.
In order to avoid this, I made the Server not to exit when cert files are not exists.
Is this feature not necessary?

Reply:
I made the same comment twice by mistake.
https://github.com/cybozu-go/sabakan/pull/270#discussion_r1365174540

This behavior makes it harder to notice these breaking changes and misconfigurations for users.
So I think, we should strictly check for preconditions by default.